src/transforms/onehot.jl

```diff
@@ -47,7 +54,9 @@ function applyfeat(transform::OneHot, feat, prep)
   end
 
   newnms, newcols = first.(onehot), last.(onehot)
-
+  if transform.categorical == true
+    newcols = [categorical(newcol; levels=nothing, ordered=false, compress=false) for newcol in newcols]
```

Comment:
Do we need to pass all these keywords explicitly? Isn't a simple `categorical(newcol)` enough?

Reply:
Yes `categorical(newcol)` is enough, but I wanted to explicitly mention the parameters for the first time to make it easier for anyone going through the code.

Reply:
Please remove all options as they don't help the reader of the code.